spec.html

```diff
@@ -1016,6 +1016,117 @@ <h1>Example Cyclic Module Record Graphs</h1>
+    </ins></p>
+
+    <emu-table id="table-module-graph-cycle-async-fields-7" caption="Module fields after module _C_ finishes with an error">
+      <table>
+        <thead>
+          <tr>
+            <th>Module</th>
+            <th>[[DFSIndex]]</th>
+            <th>[[DFSAncestorIndex]]</th>
+            <th>[[AsyncEvaluating]]</th>
+            <th>[[AsyncParentModules]]</th>
+            <th>[[PendingAsyncDependencies]]</th>
+            <th>[[EvaluationError]]</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th>_A_</th>
+            <td>0</td>
+            <td>0</td>
+            <td>*true*</td>
+            <td>&laquo; &raquo;</td>
+            <td>1 (_B_)</td>
+            <td>~empty~</td>
+          </tr>
+          <tr>
+            <th>_C_</th>
+            <td>2</td>
+            <td>1</td>
+            <td>*false*</td>
+            <td>&laquo; _A_ &raquo;</td>
+            <td>_C_'s evaluation error</td>
+          </tr>
+        </tbody>
+      </table>
+    </emu-table>
+
+    <p><ins>
+      _A_ will be rejected with the same error as _C_, as _C_ will call AsynModuleExecutionRejected on _A_, with _C_'s error. _A_.[[AsyncEvaluating]] is set to *false*.  At this point, the Promise in _A_.[[TopLevelCapability]] (which was returned from _A_.Evaluate()) is rejected. The fields of the updated module are as given in <emu-xref href="#table-module-graph-cycle-async-fields-8"></emu-xref>.
+    </ins></p>
+
+    <emu-table id="table-module-graph-cycle-async-fields-8" caption="Module fields after module _A_ is rejected">
+      <table>
+        <thead>
+          <tr>
+            <th>Module</th>
+            <th>[[DFSIndex]]</th>
+            <th>[[DFSAncestorIndex]]</th>
+            <th>[[AsyncEvaluating]]</th>
+            <th>[[AsyncParentModules]]</th>
+            <th>[[PendingAsyncDependencies]]</th>
+            <th>[[EvaluationError]]</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th>_A_</th>
+            <td>0</td>
+            <td>0</td>
+            <td>*false*</td>
+            <td>&laquo; &raquo;</td>
+            <td>0</td>
+            <td>_C_'s Evaluation Error</td>
+          </tr>
+        </tbody>
+      </table>
+    </emu-table>
+
+    <p><ins>
+      Then, _B_ finishes executing without an error. When that happens, AsyncModuleExecutionFulfilled is called again and _B_.[[AsyncEvaluating]] is set to *false*.  However, _A_ has already been rejected, so it will not be executed, and the sortedExecList will be empty, and AsyncModuleExecutionFulfilled will return. The fields of the updated modules are as given in <emu-xref href="#table-module-graph-cycle-async-fields-9"></emu-xref>.
```

Comment:
`sortedExecList` → `_sortedExecList_`

Reply:
Not sure if this goes too much into the guts of `GatherAsyncParentCompletions`, but thought it worth mentioning:

```suggestion
      Then, _B_ finishes executing without an error. When that happens, AsyncModuleExecutionFulfilled is called again and _B_.[[AsyncEvaluating]] is set to *false*. GatherAsyncParentCompletions is called on _B_. However, _A_.[[CycleRoot]] is _A_ which has an evaluation error, so it will not be added to the returned _sortedExecList_ and AsyncModuleExecutionFulfilled will return without further processing. The fields of the updated modules are as given in <emu-xref href="#table-module-graph-cycle-async-fields-9"></emu-xref>.
```

It could also be worth adding a line here about how future rejections pass through `[[CycleRoot]]`:

> Any future importer of _B_ will resolve the rejection of _B_.[[CycleRoot]].[[EvaluationError]] from the evaluation error from C that was set on the cycle root _A_.